.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build Timecop
+on:
```

Comment:
You may also want `workflow_dispatch` so that you can manually trigger builds inside the Actions UI

Reply:
Don't really see a need for it right now.

Reply:
Fair enough just a thought